packages/notification-system/src/assets/icons.js

```diff
@@ -0,0 +1,31 @@
+export const Close = () => (
+  <svg
+    width="27"
+    height="27"
+    viewBox="0 0 27 27"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M1 1L13.5 13.5M26 26L13.5 13.5M13.5 13.5L26 1L1 26"
+      stroke="#1572A1"
+      stroke-width="2"
+    />
+  </svg>
+);
+
+export const Spinner = () => (
```

Comment:
Let's add one component per file, this favor the encapsulation

Reply:
Fixed